src/RadioGroup/RadioGroup.tsx

```diff
@@ -57,11 +68,11 @@ export default class RadioGroup<
         onChange={this._handleNavChange}
       >
         <div
-          aria-labelledby={this._labelID}
+          aria-labelledby={labelID}
           className={classnames(cssClass.CONTAINER, className)}
           role="radiogroup"
         >
-          <div className={cssClass.LABEL} id={this._labelID}>
+          <div className={cssClass.LABEL} id={labelID}>
```

Comment:
I think we don't want to display this `div` in the case that we're using `aria-labelledby`? Otherwise it seems like we're going to end up with two elements with the same `id` (since we're giving this `div` the `id` of `ariaLabelledBy`) which seems like an issue.

Reply:
ah good catch! fixed! I also added some mechanisms to prevent label and ariaLabelledBy from being passed in together, and made sure to keep it backward compatible. added an example in the documentation too